src/views/darkroom.c

```diff
@@ -273,7 +274,11 @@ void _display_module_trouble_message_callback(gpointer instance,
 
     dt_iop_gui_update_header(module);
 
-    if(label_widget) gtk_widget_destroy(label_widget);
+    if(label_widget)
+    {
+      GtkWidget *iopw = gtk_widget_get_parent(module->widget);
+      gtk_container_remove(GTK_CONTAINER(iopw), label_widget);
```

Comment:
https://developer.gnome.org/gtk3/stable/GtkWidget.html#gtk-widget-destroy
When a widget is destroyed all references it holds on other objects will be released:
- if the widget is inside a container, it will be removed from its parent